src/subdomains/generic/kyc/dto/input/kyc-data.dto.ts

```diff
@@ -148,15 +148,40 @@ export class KycBeneficialData {
   @IsBoolean()
   hasBeneficialOwners: boolean;
 
+  @ApiProperty({ description: 'Is the account holder a beneficial owner?' })
+  @IsNotEmpty()
+  @IsBoolean()
+  accountHolderIsBeneficialOwner: boolean;
+
+  @ApiPropertyOptional({ type: BeneficialOwnerData, isArray: true })
+  @ValidateIf((d: KycBeneficialData) => !d.hasBeneficialOwners && !d.accountHolderIsBeneficialOwner)
+  @IsNotEmpty()
+  @IsArray()
+  @ValidateNested({ each: true })
+  @Type(() => BeneficialOwnerData)
+  managingDirector: BeneficialOwnerData;
```

Comment:
Maybe we should rename `BeneficialOwnerData` to something more generic?

Reply:
Hm yeah do you have an idea? 😁

Reply:
Maybe `OrganizationMemberData`?

Reply:
ContactPersonData or something like that? 🤔

Reply:
It's all about users linked to an organization, right?

Reply:
Yeah so at this point it's just managing director and shareholder but the Dto is neutral so it can be a neutral name 👍